packages/webpack/preset.js

```diff
@@ -30,6 +30,13 @@ module.exports = {
       mainFields: ['mixin:server', 'mixin:runtime', 'mixin'],
     },
   },
+  browserWhitelist: {
+    host: true,
```

Comment:
What about nested keys?
For example, if the config looks like this:
```json
{
  "untool": {
    "basePath": "/",
    "something": {
      "someSecret": "mysecret",
      "someSafeValue": "this-is-needed-in-the-browser"
    },
    "browserWhitelist": {
      "basePath": true
    }
  }
}
```

We could either replicate the nested object in the white-list or we could use something like the dot-notation and have the nesting entirely in the key (eg. `"something.someSafeValue": true`).

Reply:
We decided to go with dot notation.